tests/meta/test_ensure_type_hints.py

```diff
@@ -30,7 +30,7 @@ def pytest_generate_tests(metafunc: _pytest.python.Metafunc) -> None:
             if module.startswith("gitlab.v4.objects"):
                 excluded_modules.add(module)
```

Comment:
This might be unrelated here but for more speed-up we can also remove the entire `toml` parsing section here now I think (I can't comment higher up in the diff).

Reply:
True. Good point. I'll do that too.